src/applications/gi/components/vet-tec/VetTecFilterBy.jsx

```diff
@@ -54,6 +106,8 @@ class VetTecFilterBy extends React.Component {
           onChange={this.handleFilterChange}
           options={options}
         />
+        {/* prod flag for CT-116 - #19992 */}
+        {!environment.isProduction() && this.renderProviderFilters()}
```

Comment:
@annaswims Sorry for the ping in Slack and here on this, but what's the current recommendation for feature toggles? Should we still be using a build-type check, or are the feature toggles ready for VFS teams yet?

Reply:
Can I get an update on this? We're ready to merge this ASAP and will be removing the prod flag soon.

Reply:
Feature toggles are ready for all teams and are documented on this page https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/tools/feature-flags/  starting at "Release Toggles"

Reply:
This is waiting on https://github.com/department-of-veterans-affairs/vets-api/pull/3605

Reply:
@cvalarida @annaswims now that the feature flag in vets-api, is this waiting on anything else?